docs/examples/provision-with-terraform.md

```diff
@@ -0,0 +1,220 @@
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that the cross-cluster network tunnel has been established, and an outgoing peering is currently active (i.e., the *Rome* cluster can offload workloads to the *Milan* one, but not vice versa):
+
+```text
+NAME    TYPE        OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+milan   OutOfBand   Established        None               Established   Established      12s
+```
+
+At the same time, you should see a virtual node (`liqo-milan`) in addition to your physical nodes:
+
+```bash
+kubectl get nodes
+```
+
+```text
+NAME                 STATUS   ROLES                  AGE     VERSION
+liqo-milan           Ready    agent                  14s     v1.23.6
+rome-control-plane   Ready    control-plane,master   7m56s   v1.23.6
+rome-worker          Ready    <none>                 7m25s   v1.23.6
+```
+
+### Offload a namespace
+
+If you want to deploy an application that is scheduled on a Liqo virtual node (hence, it is offloaded on a remote cluster), you should first create a namespace where your pod will be started.
+
+This can be achieved with `kubernetes_namespace` of [kubernetes](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs) Terraform provider, configured with either the config file or the full list of parameters of the local cluster.
+
+Then tell Liqo to make this namespace eligible for the pod offloading.
+
+The resource in charge of doing this is `liqo_offload` of the same `liqo` Terraform provider instance of `liqo_peering` resource:
+
+```terraform
+resource "liqo_offload" "offload" {
+
+  depends_on = [
+    helm_release.install_liqo_rome,
+    kubernetes_namespace.namespace
+  ]
+
+  provider = liqo.rome
+
+  namespace = "liqo-demo"
+
+}
+```
+
+```{admonition} Note
+Liqo virtual nodes have a [taint](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) that prevents the pods from being scheduled on them.
+The Liqo webhook will add the toleration for this taint to the pods created in the liqo-enabled namespaces.
+```
+
+Since no further configuration is provided, Liqo will add a suffix to the namespace name to make it unique on the remote cluster (see the dedicated [usage page](/usage/namespace-offloading.md) for additional information concerning namespace offloading configurations).
+
+You can now test the infrastructure you have just created by deploying an application.
+For this, you can follow the [proper example section](StartHelloWorldApplication) in the Quick Start page.
+
+## Tear down the infrastructure
+
+To tear down all the infrastrucure you only need to run the following command:
+
+```bash
+terraform destroy
+```
+
+This command will destroy all the resource starting from the last one created, from bottom to top.
+
+If you want to destroy a specific resource (for example to un-peer or to un-offload) you can leverage on the `-target` flag of `destroy` command.
```

Comment:
```suggestion
If you want to destroy a specific resource (for example to unpeer a cluster or to unoffload a namespace) you can leverage on the `-target` flag of `destroy` command.
```

Just for consistency, as in the rest of the documentation they are written without an hyphen.